src/main/java/frc/robot/subsystems/climber/ClimberIO.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.climber;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import com.ctre.phoenix6.controls.TorqueCurrentFOC;
+
+/** Generic subsystem hardware interface. */
+public interface ClimberIO {
+  /** Contains all of the input data received from hardware. */
+  @AutoLog
+  public static class ClimberIOInputs {
+    double leftVelocityRPS = 0.0;
+    double leftPositionMeters = 0.0;
+    double leftStatorCurrentAmps = 0.0;
+    double leftSupplyCurrentAmps = 0.0;
+    double leftCurrentSetpoint = 0.0;
```

Comment:
maybe just call this setpoint as it may be position or current depending on the request made